test/unit/adapters/http.js

```diff
@@ -21,12 +21,9 @@ describe('supports http with nodejs', function () {
       proxy.close();
       proxy = null;
     }
-    if (process.env.http_proxy) {
-      delete process.env.http_proxy;
-    }
-    if (process.env.no_proxy) {
-      delete process.env.no_proxy;
-    }
+    delete process.env.http_proxy;
+    delete process.env.https_proxy;
+    delete process.env.no_proxy;
```

Comment:
Two small changes here:
1) Removed unnecessary `if` (I can revert if you think it was better with it)
2) Moved here the deletion of `process.env.https_proxy`, which was manually removed in a finally in a test that used it. It felt better to have this centralized here.